sh/tmuxcomplete

```diff
@@ -27,7 +27,8 @@
 # Words including 2000 lines of history per pane
 #     sh tmuxcomplete -c '-S -2000'
 
-if ! tmux info &> /dev/null; then
+tmux info &> /dev/null
+if [ "$?" != "0" ]; then
```

Comment:
`!=` also is not POSIX, it requires bash double-brackets `[[ ]]`. POSIX would be:

    if ! [ "$?" = "0" ]; then

or I would tend to use:

    if [ $? -ne 0 ] ; then